core/domain/feedback_services.py

```diff
@@ -113,6 +113,9 @@ def create_message(
     # added to it.
     if message_id != 0 and (updated_status or updated_subject):
         if updated_status and updated_status != thread.status:
+            add_feedback_thread_status_change_email(
```

Comment:
Note again that it's possible to add a message at the same time as adding a status change.